app/src/main/java/com/github/dedis/student20_pop/ui/CameraPermFragment.java

```diff
@@ -0,0 +1,73 @@
+package com.github.dedis.student20_pop.ui;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.app.ActivityCompat;
+import androidx.fragment.app.Fragment;
+
+import com.github.dedis.student20_pop.R;
+
+/**
+ * Fragment handling permission granting for the camera
+ */
+public final class CameraPermFragment extends Fragment implements View.OnClickListener {
```

Comment:
in CameraPermFragment, Perm stands for Permission ? I would not be against CameraPermissionFragment as name

Reply:
I agree with this

Reply:
Done